### ISSUE TITLE
Get our tool listed on YNAB website

### ISSUE THREAD

**Issue by torbengb** [torbengb/bank2ynab#50]
Specifically at the end of these pages: 

- [YNAB 4 support page](https://classic.youneedabudget.com/support/article/csv-file-importing)
- [nYNAB help page](https://docs.youneedabudget.com/article/173-file-based-import)

**Comment by nocalla** [comment]
Will they do that for a third party tool? Do they reference the toolkit
anywhere?

**Comment by torbengb** [comment]
It wouldn't hurt to ask. The YNAB 4 page linked above lists several CSV tools, and they seem to be in various states of disrepair.

**Comment by nocalla** [comment]
Worth a try. No point going there until we have the bugs out though.

On 23 Oct 2017 15:51, "Torben Gundtofte-Bruun" <notifications@github.com>
wrote:

It wouldn't hurt to ask. The YNAB 4 page linked above lists several CSV
tools, and they seem to be in various states of disrepair.

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/torbengb/bank2ynab/issues/50#issuecomment-338684907>,
or mute the thread

**Comment by nocalla** [comment]
Interestingly, there is a 3rd Party Github project listed on the [official YNAB site](https://docs.youneedabudget.com/article/921-importing-a-csv-file) now. 

It doesn't work all that well on our test data, but it's more user-friendly in that it's drag n' drop onto a [web page](https://aniav.github.io/ynab-csv/).

**Comment by torbengb** [comment]
Interesting! So we would probably have a chance to get listed there too. I'll make another contact to YNAB.

I'm a bit confused and haven't tried out either of the two yet, but it seems that https://github.com/aniav/ynab-csv is forked from https://github.com/halloffame/ynab-csv and I wonder what the differences are.

Both of those tools have a step 3 saying *"For each column in the YNAB data file, choose which column you want to pull from your source data file"* and that sounds like the spot where we have a leg up?

**Comment by nocalla** [comment]
Yeah, their one takes more effort per conversion, because you have to drag
things around a bit. Ours has a higher barrier of entry though, requiring
python installation and config file editing.

**Comment by torbengb** [comment]
I'm submitting this message:

Subject:  
Website: link to free CSV conversion tool

Hi YNAB team!
I've seen that your website mentions CSV conversion tools here[1] and here[2]. Unfortunately, these require several steps for the user to do for every CSV file. 

Luckily, some enthusiasts (including myself) have made "bank2ynab": a simple one-click tool that automatically detects the bank export file from more than 20 banks worldwide (and growing).

--> It would be great for your users, and I would be deeply thankful, if you would add a link to "bank2ynab" from the two pages where you already have links. 

Please use this link:
https://github.com/torbengb/bank2ynab#bank2ynab

Thank you very much!
Best regards from Austria,
Torben

[1] https://docs.youneedabudget.com/article/921-importing-a-csv-file
[2] https://classic.youneedabudget.com/support/article/csv-file-importing

**Comment by nocalla** [comment]
Maybe include that it requires a bit of tech savvy and installing python.

**Comment by torbengb** [comment]
I'd rather not - prefer to keep this short and direct. Interested users would have to see the Readme anyway, and it's clear enough in there.

**Comment by nocalla** [comment]
Okay, fair point. Fire away!

**Comment by torbengb** [comment]
I'm waiting for the pull request to pass first, just to make sure they're not going to check, and seeing old code.

**Comment by torbengb** [comment]
Their response:

> Thanks for bringing attention to your tool! Right now we're getting ready to launch a public API for YNAB, which I think your tool could benefit from :) If you think so, too, I'd love for you to [fill out this form](https://goo.gl/forms/Lm3sl7TUOpeBUg0i2) to request early access. 

> Because of the API release, I don't think we'll be officially recommending any other tools on the website in that manner - we'll likely have some other place for 3rd-party tools to be collected. 

> Thank you so much for the work you've done to create the converter!

So we're not getting listed, it appears. :cry: I've signed us up for the early access - it certainly can't hurt, and let's see how it might be useful to us!

FYI @nocalla

**Comment by nocalla** [comment]
I wonder what the API consists of.
